src/components/common/ColonyMembers/MembersFilter/MembersFilter.tsx

```diff
@@ -86,7 +86,7 @@ const MembersFilter = ({ onFilterChange, isRootOrAllDomains }: Props) => {
                         name={name}
                         options={options}
                         label={label}
-                        handleFilterChange={onFilterChange}
```

Comment:
Like here for example @jakubcolony. What would be the correct behaviour?

The component wraps our Select component, which exposes an `onChange` prop.

Reply:
It seems a bit odd that currently the handle is being defined as `handle`, then passed as `on`, then being passed again as `handle`

Reply:
```js
onFilterChange={onFilterChange}
```

Since both are props